### PR TITLE
Frontfixes

### PR DESCRIPTION
**Pull request by Arnanasas**
fixed front-end navbar, deleted old one, connected with auth0 authentication service

### CODE REVIEW

File: WebUI/src/components/Navbar.tsx [hunk 1]
```diff
@@ -72,11 +70,30 @@ const NavigationBar = () => {
...
+                <Link to='/addlisting' className='nav-link'>
+                  Add Listing
+                </Link>
+              </StyledNavLink>
+            )}
           </Nav>
           <Nav id='nav-right'>
-            <SignInBtn>Sign In</SignInBtn>
+            {/* <SignInBtn>Sign In</SignInBtn>
+             */}
```
Comment (aauga) [inline WebUI/src/components/Navbar.tsx:95]: remove this comment

File: WebUI/src/App.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const App = () => {
 
   return (
     <div id='app' className='d-flex flex-column h-100'>
-      <Header />
+      {/* <Header /> */}
```
Comment (aauga) [inline WebUI/src/App.tsx:22]: remove this comment

**Comment by aauga** [comment]
maybe lets put the buttons and navbar into one folder (f.e. components/navbar)? would make it easier to navigate

**Comment by Arnanasas** [comment]
will be updated in upcoming pull request. thank you for you comments

**Comment by aauga** [comment]
alright, will merge this and wait for your next PR!

**Pull request merged**